### PR TITLE
Add Simple Load Balancing Util

### PR DESCRIPTION
**Pull request by gcapalbo**
@NoahCarnahan @snopoke code review buddies

### CODE REVIEW

File: dimagi/utils/load_balance.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    load_balance('outbound-phone-number', ['16175550001', '16175550002', '16175550003'])
+    """
+
+    # We need access to the raw redis client because calling incr on
+    # a django_redis RedisCache object raises an error if the key
+    # doesn't exist.
+    client = get_redis_client().client.get_client()
+
+    # Increment the key. If they key doesn't exist (or already expired),
```
Comment (snopoke) [inline dimagi/utils/load_balance.py]: spelling: `they` -> `the`

**Comment by snopoke** [comment]
:+1:  once build passes. Failure doesn't seem related so kicked the build

File: dimagi/utils/load_balance.py (hunk 1 above)
Comment (gcapalbo) [inline dimagi/utils/load_balance.py]: good catch, updated

**Comment by gcapalbo** [comment]
looks like the failure was real but should be fixed now (import error, see my comment in the latest commit)

**Pull request merged**